internal/compress/gob.go

```diff
@@ -19,18 +19,21 @@ package compress
 
 import (
 	"bytes"
-	"encoding/gob"
 	"io"
 	"reflect"
 
+	"github.com/vdaas/vald/internal/compress/gob"
 	"github.com/vdaas/vald/internal/errors"
 )
 
 type gobCompressor struct {
+	transporter gob.Transporter
```

Comment:
sorry but I am not sure what is the meaning of transporter here :|

Reply:
sorry. I think so too.
I will think about naming.